src/app/shared/image/image-upload.component.spec.ts

```diff
@@ -8,6 +8,68 @@ describe('ImageUploadComponent', () => {
+    3BpZb9o/6g39RUvS8XIRYFYA8bT/jFUwvxFwLNxG4fzConRC9EHMt7CTo+NSAdjp0KO2boVDIBOtxAxUadUV5vs+vrkg6lPU127djBmoJ+RavQDovhX\
+    ykvQKjzlmmWbGcR8iR+gq81kE+oigvroGsEDpsPZ6pHPPAcgp0WEWeqRzzrsByAnQoTC6JnFdg1Esgn9EQXE6ADnKhKs+EkJef3nSucsxbGxF504DXn\
+    6PcdJCLiZCzi8fsy3ik6PWKMYcq754UJv9M7qZM58oxL8iRr7iqTj5MmK5+V4P6oh60cmBVLC2n0KWXIOZCtuSwlYZVaSEI07n6de4kvyxtLVWSpcM2\
+    HMX7uQHydlQeg4wHXTrIR7SJkT8eszKbZOk6rx8xlErR4X9nETlnN+blZOm6Xt+sLO/IofK6QhUKUbolF2pyTXagMX+m7svqT5r0vFxpo5EV83asouD\
+    VtafqXknSrbpXVbnW126voTHZ61/JSdIteUJNrmkddjmUiV27+XStM0G6WQK50ZHjiPzPo2dUedOjg7xVUW4YHSlM/sj2M+vbydGtil/lmPch8iH2z3\
+    FycnQzX+WtclvTNNGRslcCxtmji8fvAiBGh5jPmCD5puGi/Q+06CbPTEzMJ7OLu4t3fpCiW3JDVZ5Fqj9BvtC+54UQHao/Z6jLM6h8rn2nEyG6yRpbN\
+    TW5YdU663b5eYicEN2SS0NNbkDM/crp0EE+Q6pOU1sCyMnQQd6lKtdaMlz3d50TopdALiw5t8s3LXQ4d4AGPS9XS01Avr0hkJwGvSBXrBDqQVaR42y9\
+    o5wEPV8VpiYXshWRJ9n6budTNgjQSyNHaiaG2N+HXc4XKT8dDrmbqVgPh8ccuvs59tMlCdEnTk6dDlVh5yrXQLbitVG06QW5VL7OK49eGnkDqzw6rCJ\
+    vU5b3OFYCEqZD5cA25dqoHudKQMpR71ynLnepBCRMB7lqVViPWyUgXXrnCmV5ttGt6I8sHeSqvT2bci13pEovhTznXuhJlN60QrkG0vkkHcJ0XhJ5u9\
+    cvJkiHF0f+RrkersNTTpAOMV+nXAPZ0e59Dio5Old+kgF5X5u3nBwdKgHXKVcC+pJTo4N821mSE6PrIFeuBJzuT06LrutN54qzJSdFh0rAmcJQq5ORu\
+    9pYklUa3ZKr1kZZMfcpJ0SH817PqpwOnbO2kbRqDeSuNu5bToeeZL/frDS4Q63J9PYAcjp0nU0akwpjHNRA7gokJ3St62zjtPB2kE/PBZJTGuGH2PXT\
+    wr5eAOqLgspJ3deH2NKQdkue0oPJaT3NJdnSJWHsUA8XXE7sGT7JVoewF2Jusoqmg/2hwHaohwsjJzdfN1n/kmA5moI8RO6TXJYGjnCdFshuGH2Nod4\
+    OTS83p1tHdYtAch4q1V9uOhIveNWWf7vQOjLh5OWPuulgT/iXh3wNfHnp8OD+BjLL1NmAP7uQ2VCnqhCgx9m9u3dgb03zZ1eSl53+njz6rpNdeMsbws\
+    vLTj9HygV7UPs9XnFPyBYVednpW+Wok31gzNUuFOUEol6XkAuuQH4oH3cxYTGnQI+BD7Uzt+s9f52rNQp0iOCcD9G4O47zJZDToFv2nUjv5U7PdbCKn\
+    GHRoFv2K/3bQa4cczJ0yz7brz1hyXlk6OC5oA3xIHMZeI1OpgRyOnTwXTAbtRfN30HeWAo5Ibplb0PzNquXjHvfZkeqJHJK9EKfx+xLT9sh5rnSyEnR\
+    LTvW58E+VrBDPRycJ1MaOS062PdiY12SbRyz9nsV5KXaD0+LDrF1sO+DsQ4qhGD9vGSHIBCjO/X5JFtjvVrPd51MJdKd7ENszSdy+uwSyunRoc/Pwe0\
+    H3/pDKeUE6QW7jqWtdc6iTS/MZRB7sqRyknQr7jsm/oBxknQrb7OD61VJPyt2onT4xQt2crMq6XDs0PKdPFmNdMu+4MMJtZebfqec5/CK6bo67ehszq\
+    sw6vm24dcTeI8rL91ks9bd6fiW6fdeu/eByNLL2spNNye7vmdajzC9iqNeo9foNXqNXqPX6DV6jV6j1+g1eo1eo9foNXqNXqPX6DV6jV6j1+g1eo0eh\
+    v4xS+o0WydssqtFPSTd9RCdmDyx7+NJNNvBgT4ou3Whr/SiL5PRbJp8yYt+yP0ToDCEalvrsU38ebgmXNuTEY26FPdBTbJrezOq/X30Wa9j+7ZElb5w\
+    vRd9UVTpd13kVX4/Kar0p3Z4Ve0lD0d0mHvOa4BnelZqUQz69v3eh5O+H036Lwe9H8FTi40o0qd53dWti315FMMe2+rnMN4XFF/QQjLoV73rZ39V/ED\
+    0wp7Y4u8E5jVGImpBf7rJ9FWKP9QctS6/9hq/GykHtqejRT/p+8Bx/ltIRUWou8876D+tFv+T60aOSruxPRFk33D7ARmZPm/cFiyVOngkInbYYDcUaJ\
+    8VZw/crUXBnpC9VwY+COSBTyNwvSfk5a8ElnM2uLzSx3l40dBtYQ5/4Sy3QsZEJcM1+UQ81LE3nJkvdiu+L7yc45shn5rFwh/4c09XWjO0ytPDGfPaT\
+    ftnq63irblZVpxdA3nsjitUFzA5f+HyhRU2i4XpeVd/KVZvuZl6P3u4YvSaPLTlV52lWwrnyX8s2vLmk8eWUf4DJG7vfafnzhP9IUe2/wMBa50QI2mv\
+    NwAAAABJRU5ErkJggg==';
+  function dataURItoBlob(dataURI) {
+    var binary = atob(dataURI.split(',')[1]);
+    var array = [];
+    for(var i = 0; i < binary.length; i++) {
```

Comment:
teeny things: my linter is mad about these three `var`s plus missing whitespace between `for` and first parentheses